### PR TITLE
PPLUS-1285: Change the check for unique names of sub entities

### PR DESCRIPTION
**Pull request by VladislavStrelchenko**
Ticket: https://spryker.atlassian.net/browse/PPLUS-1285

ChangeLog: 
* Fixed return code form Evaluate command
* Allow not prefixed sub-entities if a parent is unique on project level

### CODE REVIEW

File: src/CodeCompliance/Domain/Checks/NotUnique/DatabaseColumn.php [hunk 1]
```diff
@@ -37,26 +37,36 @@ public function getViolations(): array
...
+                    return !in_array($column, $coreSchemas[$sourceName] ?? []) && !$this->hasProjectPrefix($column, $projectPrefixes);
+                },
+            );
+            $isDbPrefixExist = $this->hasProjectPrefix($sourceName, $projectPrefixes);
+            $isDbUniqueOnTheProjectLevel = !in_array($sourceName, $namesFromCoreSchemas);
 
-            if ($columnsWithoutPrefix !== [] && $columnsWithoutPrefix !== null && !$isDbPrefixExist) {
+            if (
+                $columnsWithoutPrefix !== []
+                &&
```
Comment (alexanderM91) [inline src/CodeCompliance/Domain/Checks/NotUnique/DatabaseColumn.php:63]: ++

File: src/CodeCompliance/Domain/Checks/NotUnique/TransferProperty.php [hunk 2]
```diff
@@ -36,29 +36,38 @@ public function getViolations(): array
...
+                function (string $property) use ($projectPrefixes, $transferName, $coreSchemas) {
+                    return !in_array($property, $coreSchemas[$transferName] ?? []) && !$this->hasProjectPrefix($property, $projectPrefixes);
                 },
             );
             $isTransferPrefixExist = $this->hasProjectPrefix($transfer->getName(), $projectPrefixes);
+            $isTransferUniqueOnTheProjectLevel = !in_array($transfer->getName(), $nameFromCoreTransfers);
 
-            if ($propertiesWithoutPrefix !== [] && $propertiesWithoutPrefix !== null && !$isTransferPrefixExist) {
+            if (
+                $propertiesWithoutPrefix !== []
```
Comment (alexanderM91) [inline src/CodeCompliance/Domain/Checks/NotUnique/TransferProperty.php:63]: ++

**Comment by alexanderM91** [comment]
Please add a change log.

File: src/CodeCompliance/Domain/Checks/NotUnique/TransferProperty.php (hunk 2 above)
Comment (VladislavStrelchenko) [reply src/CodeCompliance/Domain/Checks/NotUnique/TransferProperty.php]: done

**Comment by alexanderM91** [comment]
@VladislavStrelchenko,
please adjust change log.

**Pull request merged**